docs/references/Test_Data_Builders.md

```diff
@@ -2,207 +2,205 @@
 
 ![IMAGE](../images/GrowingObjectOrientedSoftwareGuidedByTests.PNG)
 
-[Copyright:[Growing Object-Oriented Software, Guided by Tests](http://www.informit.com/store/growing-object-oriented-software-guided-by-tests-9780321503626), 
+[Copyright:[Growing Object-Oriented Software, Guided by Tests](http://www.informit.com/store/growing-object-oriented-software-guided-by-tests-9780321503626),
 by [Nat Pryce](http://natpryce.com/bio.html) and [Steve Freeman](https://www.higherorderlogic.com/)]
 
 ## Summary
 
-'Test Data Builders' is a technique that leverages on the [Builder Pattern](https://en.wikipedia.org/wiki/Builder_pattern#Java) 
+'Test Data Builders' is a technique that leverages on the [Builder Pattern](https://en.wikipedia.org/wiki/Builder_pattern#Java)
 to construct complex objects in tests.  
-With 'Test Data Builders' we can omit fields or properties that do not 
+With 'Test Data Builders' we can omit fields or properties that do not
 contribute to the behavior of the object being tested.  
-A test data builder class has the following features: 
+A test data builder class has the following features:
 
-1. A field for each constructor parameter 
+1. A field for each constructor parameter
 1. The fields are initialized to a default safe value
 1. Fluent public methods to override the default values
-1. A 'build' method that returns an instance of object initialized with 
-the fields' values. 
+1. A 'build' method that returns an instance of object initialized with
+the fields' values.
 
 ## Examples
 
 Below is an example of a TestBuilder for the Country object from our code:
 
 ```java
-    import static com.murex.tbw.domain.country.Currency.US_DOLLAR;
-    import static com.murex.tbw.domain.country.Language.ENGLISH;
-    
-    public class CountryTestBuilder {
-        private String name = "";
-        private Currency currency = US_DOLLAR;
-        private Language language = ENGLISH;
-    
-        public static CountryTestBuilder aCountry() {
-            return new CountryTestBuilder();
-        }
-    
-        public CountryTestBuilder withName(String name) {
-            this.name = name;
-            return this;
-        }
-    
-        public CountryTestBuilder withCurrency(Currency currency) {
-            this.currency = currency;
-            return this;
-        }
-    
-        public CountryTestBuilder withLanguage(Language language) {
-            this.language = language;
-            return this;
-        }
-    
-        public Country build() {
-            return new Country(name, currency, language);
-        }   
+import static com.murex.tbw.domain.country.Currency.US_DOLLAR;
```

Comment:
If I am not mistaken, we need to keep this indentation for the GitHub pages to work.

Reply:
Done